Sources/Props/Toggle/GravityToggleStyle.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GravityToggleStyle.swift
+//  
+//
+//  Created by Djuro on 1/11/22.
+//
+
+import SwiftUI
+
+public struct GravityToggleStyle: ToggleStyle {
+    
+    // MARK: - Properties
+    @Environment(\.primaryColor) var primaryColor: Color
+    private let onLabel: String
+    private let offLabel: String
+    
+    // MARK: - Initializer
+    public init(onLabel: String = "ON", offLabel: String = "OFF") {
+        self.onLabel = onLabel
+        self.offLabel = offLabel
+    }
+    
+    // MARK: - ToggleStyle
+    public func makeBody(configuration: Self.Configuration) -> some View {
+        GeometryReader { geometry in
+            let offset = geometry.size.width/2 - 3 * geometry.size.height/2 + 10
```

Comment:
Maybe you have some better math for this. I don't like this magic 10, but it seems that it's working 🙏

Reply:
It's looking great! I recommend making the size fixed, roughly the size of the built-in toggle, but maybe a little longer to give the effect a little more oomph. If you fix the size, you shouldn't need geometry reading which will simplify, you can decide on the size, then know how big the magnitude of the animation. 

Also as a UX thing, it'd be great if the whole thing was tappable, not just the circle.